src/pongstar.h

```diff
@@ -4,7 +4,9 @@
 
 #include <vector>
 #include <queue>
+#include <stack>
 #include <chrono>
```

Comment:
`stack` need. Our game state machine is now done through stack. `queue` don't need,